sale_order_note_template/models/sale_terms_template.py

```diff
@@ -0,0 +1,68 @@
+# Copyright 2014 Guewen Baconnier (Camptocamp SA)
+# Copyright 2013-2014 Nicolas Bessi (Camptocamp SA)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import fields, models
+
+
+class SaleTermsTemplate(models.Model):
+    _name = "sale.terms_template"
+    _description = "Sale terms template"
+
+    active = fields.Boolean(default=True)
+
+    name = fields.Char(
+        string="Name",
+        required=True,
+    )
+
+    text = fields.Html(
+        string="Terms template",
+        translate=True,
+        required=True,
+    )
+
+    engine = fields.Selection(
+        selection=[
+            ("jinja", "Jinja"),
+            # Figure out why while saving qweb instructions there are wiped out
+            # wich makes qweb templating not usable (probably for security reasons)
+            # ("qweb", "QWeb"),
+        ],
+        string="Template engine",
+        required=True,
+        readonly=False,
+        default="jinja",
+        help="""Templating format to use:
+            * **QWeb**: https://www.odoo.com/documentation/14.0/reference/qweb.html
```

Comment:
well, for the time being, the help mention this option that seems to be disabled.